genie-core/src/main/java/com/netflix/genie/core/jpa/services/JpaJobSearchServiceImpl.java

```diff
@@ -362,4 +363,16 @@ public String getJobHost(@NotBlank final String jobId) throws GenieException {
             throw new GenieNotFoundException("No job execution found for id " + jobId);
         }
     }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public Long getActiveJobCountForUser(@NotBlank final String user) throws GenieException {
```

Comment:
Whoa, i just deleted your comment on this line (i meant to delete mine).
You were saying how i shouldn't null/empty check myself.
As per side-conversation. It would be nice to test these annotations work, but it's out of scope for this.
Removed null/empty checking from this.